framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/settings/builder/KieServerSettingsBuilder.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2017 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.cloud.api.settings.builder;
+
+import org.kie.cloud.api.settings.DeploymentSettings;
+
+/**
+ * Cloud settings builder for Kie Server.
+ *
+ * If any environment variable isn't configured by SettingsBuilder, then is used
```

Comment:
then is used default variable from application template -> then default value from application template is used.